.github/CODEOWNERS

```diff
@@ -101,13 +101,14 @@
 /core/templates/pages/collection-player-page/ @kevintab95
 
 
-# Contributor Dashboard project
-/core/controllers/contributor_dashboard*.py @sagangwee
-/core/domain/opportunity*.py @sagangwee
-/core/domain/voiceover_services*.py @sagangwee
-/core/storage/opportunity/ @sagangwee
-/core/templates/domain/opportunity/ @sagangwee
-/core/templates/pages/contributor-dashboard-page/ @sagangwee
+# Contributor Dashboard project.
+# TODO(#10779): Revert ownership to @sagangwee after 2020-09-08.
```

Comment:
@sagangwee isn't this date wrong? It's in the past.

Reply:
@seanlip Ah, woops. Updated. Thanks.